pallets/asset-manager/src/benchmarks.rs

```diff
@@ -58,6 +59,20 @@ benchmarks! {
 	verify {
 		assert_eq!(Pallet::<T>::asset_id_type(asset_id), Some(asset_type));
 	}
+
+	remove_supported_asset {
+		// does not really matter what we register
```

Comment:
TODO: rework this for being able to properly map against a benchmark with x storage items in vec